cs/v1/CS_v1.0.0.yaml

```diff
@@ -514,12 +512,133 @@ paths:
     parameters: []
 components:
   schemas:
-    Place:
-      title: Place
+    PlaceOfReceipt:
+      title: Place of Receipt
       type: object
-      description: The Location specifying where the place is located.
-      x-stoplight:
-        id: 6qncmwke0iqvq
+      description: The Location specifying where the place of receipt is located.
+      required:
+        - facilityTypeCode
+        - location
+        - dateTime
+      properties:
+        facilityTypeCode:
+          description: |
+            The code to identify the specific type of facility. The code indicates which role the facility plays during the transportCall.
+            - `BORD` (Border)
+            - `CLOC` (Customer Location)
+            - `COFS` (Container Freight Station)
+            - `OFFD` (Off Dock Storage)
+            - `DEPO` (Depot)
+            - `INTE` (Inland Terminal)
+            - `POTE` (Port Terminal)
+            - `PBPL` (Pilot Boarding Place)
+            - `BRTH` (Berth)
+            - `RAMP`  (Ramp)
+            - `WAYP` (Waypoint)
+          example: POTE
+          type: string
+        location:
+          oneOf:
+            - $ref: '#/components/schemas/UNLocationLocation'
+            - $ref: '#/components/schemas/FacilityLocation'
+            - $ref: '#/components/schemas/AddressLocation'
+          discriminator:
+            propertyName: locationType
+            mapping:
+              ADDR: '#/components/schemas/AddressLocation'
+              FACS: '#/components/schemas/FacilitySMDGLocation'
+              UNLO: '#/components/schemas/UNLocationLocation'
+        dateTime:
+          type: string
+          format: date-time
+          example: '2025-01-14T09:21:00+01:00'
+          description: 'The local date and time, when the event will take place, in ISO 8601 format specifying the offset.'
+    PlaceOfDelivery:
+      title: Place Of Delivery
+      type: object
+      description: The Location specifying where the place of delivery is located.
+      required:
+        - facilityTypeCode
+        - location
+        - dateTime
+      properties:
+        facilityTypeCode:
+          description: |
+            The code to identify the specific type of facility. The code indicates which role the facility plays during the transportCall.
+            - `BORD` (Border)
+            - `CLOC` (Customer Location)
+            - `COFS` (Container Freight Station)
+            - `OFFD` (Off Dock Storage)
+            - `DEPO` (Depot)
+            - `INTE` (Inland Terminal)
+            - `POTE` (Port Terminal)
+            - `PBPL` (Pilot Boarding Place)
+            - `BRTH` (Berth)
+            - `RAMP`  (Ramp)
+            - `WAYP` (Waypoint)
+          example: POTE
+          type: string
```

Comment:
Since we changed these properties to not be Enum any more - I think you should add a `maxLength: 4`